testing/features/support/drivers/browserstack/base.rb

```diff
@@ -65,11 +65,15 @@ def browserstack_hub_url
       end
 
       def build_name
-        PayloadValuesGenerator.build_name
+        "Design System - #{identifier} - #{base_url}"
       end
 
       def session_name
-        PayloadValuesGenerator.session_name
+        "SHA: #{sha} - CALLING_PROJECT: design-system"
+      end
+
+      def identifier
+        browserstack_build_name.present? ? browserstack_build_name : sha
```

Comment:
if you remove the present call here, and alter the original method to a `[]` graceful failure. Then it fixes any potential NME's

This then can just become `browserstack_build_name || sha`

Even better would be (Given we contain the sha in the session_name), would be to make this `browserstack_build_name || "Local Machine run - Month #{Time.now.month} - Ignore results!"`